packages/amplication-client/src/Project/ArchitectureConsole/hooks/useModelOrganizer.tsx

```diff
@@ -73,6 +76,9 @@ const useModelOrganization = ({ projectId, onMessage }: Props) => {
   const [duplicateEntityError, setDuplicateEntityError] =
     useState<boolean>(false);
 
+  const [duplicateEntityErrorMessage, setDuplicateEntityErrorMessage] =
```

Comment:
let's just call this ErrorMessage so we can use this for any future message (it is already not just used for duplicate.
Also, I don't think we need two state  - it is enough to use the "ErrorMessage" as string | null